_assets/scripts/run_functional_tests.sh

```diff
@@ -24,29 +24,36 @@ mkdir -p "${merged_coverage_reports_path}"
 mkdir -p "${test_results_path}"
 
 all_compose_files="-f ${root_path}/docker-compose.anvil.yml -f ${root_path}/docker-compose.test.status-go.yml"
+timestamp=$(date +%s)
 
 # Run functional tests
 echo -e "${GRN}Running tests${RST}, HEAD: $(git rev-parse HEAD)"
-docker-compose ${all_compose_files} up -d --build --remove-orphans
+docker-compose -p ${timestamp} ${all_compose_files} up -d --build --remove-orphans
```

Comment:
How about being a bit more concrete?
```suggestion
docker-compose -p "status-go-func-tests-${timestamp}" ${all_compose_files} up -d --build --remove-orphans
```